crates/rspack_binding_options/src/options/raw_split_chunks/mod.rs

```diff
@@ -211,7 +211,7 @@ impl From<RawSplitChunksOptions> for rspack_plugin_split_chunks::PluginOptions {
               .automatic_name_delimiter
               .unwrap_or(overall_automatic_name_delimiter.clone()),
             filename: v.filename.map(Filename::from),
-            reuse_existing_chunk: v.reuse_existing_chunk.unwrap_or(true),
+            reuse_existing_chunk: v.reuse_existing_chunk.unwrap_or(false),
```

Comment:
This should be a breaking change.

 And webpack seems to use `true` by default?

https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkscachegroupscachegroupreuseexistingchunk

![image](https://github.com/user-attachments/assets/0d88cc8a-35bc-4c26-a163-2511e330f26e)

Reply:
I think this is a ambiguous description, `reuseExistingChunk` is `true` only for `default` and `defaultVendor` cacheGroups, and `undefined` for other cache groups, you can see here https://github.com/webpack/webpack/blob/main/lib/optimize/SplitChunksPlugin.js#L779.

Reply:
Get, I also wonder why not enable reuseExistingChunk for all cacheGroups by default. Is there any reason?

Reply:
Not sure, but I guess if all cache groups are reusing existing chunk, some chunks may not meet user's expectation

Reply:
This doesn't make sense. Rspack has always enabled `reuseExistingChunk` by default, and no one seems to have reported any issues with it. Perhaps we can test the impact of this in a real project.

Reply:
This option has no actual optimization, the most differences are just output names, you can see an example here

https://github.com/webpack/webpack.js.org/issues/2122#issuecomment-388609306

Also the reuseExistingChunk will have extra check, so the perf is a little bit slow,(just a little bit

Reply:
Get, so it has no effect on the bundle size